libcxx/docs/ReleaseNotes/21.rst

```diff
@@ -39,6 +39,7 @@ Implemented Papers
 ------------------
 
 - N4258: Cleaning-up noexcept in the Library (`Github <https://github.com/llvm/llvm-project/issues/99937>`__)
+- P2542R8: ``views::concat`` (`Github <https://github.com/llvm/llvm-project/issues/105419>`__)
```

Comment:
Recent PR(s) caused conflict. Personally, I want to resolve the conflict by moving this line after P1361R2.